daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/ledger/AuthorizingTransaction.scala

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.ledger
+
+import com.daml.lf.transaction.Node.{NodeCreate, NodeExercises, NodeFetch, NodeLookupByKey}
+import com.daml.lf.transaction.{NodeId, Transaction => Tx}
+import com.daml.lf.value.Value.ContractId
+
+object AuthorizingTransaction {
+
+  private object CheckState {
+    def Empty =
```

Comment:
```suggestion
    val Empty =
```
Any reason to create a new one every time it's called?

Reply:
Nope